### ISSUE TITLE
Dependencia de models de api_mid_financiera

### ISSUE THREAD

**Issue by JuanPiedrahita** [udistrital/utils_oas#1]
Ruler_tools.go depende de "github.com/udistrital/api_mid_financiera/models", esto deberia ser global y no depender de los modelos fijados en api_mid_financiera

**Comment by miguelramirez93** [comment]
no, eso se arreglo con la elaboracion de utils_oas y se externalizo dicha dependencia

**Comment by JuanPiedrahita** [comment]
No señor, precisamente, estaba poniendo en polux lo de utils_oas y cuando importo el ruler me  manda ese error ../utils_oas/ruler/ruler_tools.go:10:2: cannot find package "github.com/udistrital/api_mid_financiera/models", es decir quedo dependiendo de los modelos de apo_mid_financiera

**Comment by miguelramirez93** [comment]
si perdon no vi el repo

**Comment by JuanPiedrahita** [comment]
todo bien,j me dijo que lo generalizaban con una interfaz que creara el issue, mientras tanto descargue el api para poder probar el utils_oas.

**Comment by miguelramirez93** [comment]
si por ahora esa es la solucion
